src/ble/tests/TestBleLayer.cpp

```diff
@@ -121,9 +129,9 @@ class TestBleLayer : public BleLayer,
 
     ///
     // Implementation of BleLayerDelegate
-
-    void OnBleConnectionComplete(BLEEndPoint * endpoint) override {}
-    void OnBleConnectionError(CHIP_ERROR err) override {}
+    // override OnBleConnectionComplete and OnBleConnectionError to set flags
```

Comment:
This comment is not needed - this is just what the code does (does  not add to reading the code, just re-reads it).

```suggestion
```